src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -102,7 +102,25 @@ def _query_multi_armed_bandit_probs(client):
   return client.query(query=BANDIT_PROBABILITY_QUERY).rows
 
 
-def _query_and_upload_strategy_weights(client):
+def _store_probs_in_bigquery(data):
+  """Update a bigquery table containing the daily updated
+  probability distribution over strategies."""
+  bigquery_data = []
+
+  for row in data:
+    bigquery_row = {
+        'strategy_name': row['strategy'],
+        'probability': row['bandit_weight'],
+        'run_count': row['run_count']
+    }
+    bigquery_data.append(big_query.Insert(row=bigquery_row, insert_id=None))
+
+  client = big_query.Client(
```

Comment:
This is making me realize that there's no reason to pass client as a variable from the handler to _query_multi_armed_bandit_probs (should that be weights as well). It was useful for the fuzzer weights code since we needed to query in multiple places, but you only use the client once so might as well instantiate it there.